### PR TITLE
Feature/brbdcf 755 integrate alien 2 kubernetes artifacts with janus

### PR DESCRIPTION
**Pull request by adidanes**
# Pull Request description

Implement features that allow deployment on K8S of TOSCA applications based on Docker containers.

- The applications uses **only** the TOSCA normative types supported by Alien 
- The application is deployed by a Yorc orchestrator on a location that contains specific resources and modifiers for deployment on a K8S cluster

Samples are provided to allow testing the deployment on K8S.

### What I did
Update alien4cloud-yorc-plugin

- Add support for Alien defined k8S resources
- Add workflow modifier for k8S that integrates call to the Alien defined modifier
- Add samples
- Update the documentation

Update K8S provider in Yorc

### CODE REVIEW

**Comment by adidanes** [comment]
Continue K8S support ...

**Comment by adidanes** [comment]
May be I made something wrong ... It seems that the PR is closed ... ?

File: data/tosca/yorc-kubernetes-types.yml [hunk 1]
```diff
@@ -12,7 +12,37 @@ artifact_types:
...
+    derived_from: tosca.artifacts.Deployment
+
 node_types:
+  yorc.nodes.kubernetes.api.types.DeploymentResource:
+    derived_from: org.alien4cloud.kubernetes.api.types.DeploymentResource
+    interfaces:
+      Standard:
+        create:
+          implementation:
+            file: "embeded"
```
Comment (laurentganne) [inline data/tosca/yorc-kubernetes-types.yml]: no issue, just a typo: embedded

**Comment by stebenoist** [comment]
You need to merge develop in order to retrieve travis config and allow passing CI job

**Comment by adidanes** [comment]
First version of K8S support using K8S artifacts provide by Alien.

**Pull request merged**